src/gc/unix/gcenv.unix.cpp

```diff
@@ -419,16 +412,15 @@ size_t GCToOSInterface::GetLargestOnDieCacheSize(bool trueSize)
 //  specify a 1 bit for a processor when the system affinity mask specifies a 0 bit for that processor.
 bool GCToOSInterface::GetCurrentProcessAffinityMask(uintptr_t* processMask, uintptr_t* systemMask)
 {
-    // TODO(segilles) processor detection
-    return false;
+    return !!::GetProcessAffinityMask(::GetCurrentProcess(), (PDWORD_PTR)processMask, (PDWORD_PTR)systemMask);
```

Comment:
GetProcessAffinityMask is defined in numa.cpp. I'm not sure how to move it here. Perhaps numa.cpp should move under gc? I will revert back to the `return false` implementation for now.